apps/ejabberd/src/mod_muc_light_admin.erl

```diff
@@ -119,9 +119,10 @@ create_unique_room(Domain, RoomName, Creator, Subject) ->
 
 invite_to_room(Domain, RoomName, Sender, Recipient) ->
     R = muc_light_room_name_to_jid(jid:from_binary(Sender), RoomName, Domain),
-    S = jid:from_binary(Sender),
+    S = jid:replace_resource(jid:from_binary(Sender), <<>>),
```

Comment:
But what worries me most is the empty resource. The resource should always be empty for the REST API. So there is no need to replace a resource with empty one.